docs/source/templates/readme.rst_t

```diff
@@ -6,15 +6,21 @@
 
 {{ about.summary | escape }}
 
-======== ===========
-Home     {{ about.home }}
-Versions {{ versions.keys() | sort | join(", ") }}
-License  {{ about.license }}
-Recipe   {{ gh_recipes }}{{ recipe_path }}
+============= ===========
+Home          {{ about.home }}
+Versions      {{ versions.keys() | sort | join(", ") }}
+License       {{ about.license }}
+Recipe        {{ gh_recipes }}{{ recipe_path }}
+{% if about.doc_url and about.doc_url != about.home %}
```

Comment:
We could `rstrip('/')` here to be slightly more reliable.

Reply:
Which one? Wouldn't it be better to use linting to enforce correct strings?